lib/rack/attack/store_proxy/redis_store_proxy.rb

```diff
@@ -4,6 +4,13 @@ module Rack
   class Attack
     module StoreProxy
       class RedisStoreProxy < SimpleDelegator
+        def initialize(*args)
+          major_version = Redis::VERSION.split('.').first.to_i
+          warn 'RackAttack requires Redis gem >= 3.0.0.' if major_version < 3
```

Comment:
Can we please use `Gem::Version` built-in version comparison instead of manually splitting the string?

Something like `Gem::Version.new(Redis::VERSION) < Gem::Version.new("3")` e.g.?